src/models/Vote.js

```diff
@@ -0,0 +1,17 @@
+const mongoose = require("mongoose");
+
+const voteSchema = new mongoose.Schema(
+  {
+    activity: {
+      type: mongoose.Schema.Types.ObjectId,
+      ref: "Activity",
+    },
+    count: {
+      type: Number,
+      default: 0,
+    },
+  },
+  { timestamps: true }
+);
```

Comment:
When we have users, we'll associate a vote with a user, so we would have one "row" or document in the `votes` collection per user vote for an activity. So the `voteSchema` would be something like this, right?

```
  {
    activity: {
      type: mongoose.Schema.Types.ObjectId,
      ref: "Activity",
    },
  {
    user: {
      type: mongoose.Schema.Types.ObjectId,
      ref: "User",
    }
}
```

We can get closer to this now by just removing the count field, and adding a new row/document to this collection for every vote for an activity. So it would just be:
```
const voteSchema = new mongoose.Schema(
  {
    activity: {
      type: mongoose.Schema.Types.ObjectId,
      ref: "Activity",
    }
}
```
Then we'd calculate the count by counting rows returned when we query on that activity id (the same way we calculate votes after we have users). What do you think of that idea?

Reply:
Yes this makes sense. I was planning to do something like this once I implemented the users model. But let's not remove this field from the Votes data model because if we do so this will break the votes controller. I'll change this on the User model PR. Thoughts?

Reply:
If you'd like to just move along for now, I'm okay with that. 👍